### DISCUSSION TITLE
Decisão: Open Source, Open-Source ou OpenSource

### DISCUSSION THREAD

**Discussion by dellagustin** [angolaosc/aosc.social#49] [Announcements]
No site, repositório e organização do GitHub vejo diferentes ocorrências desse termo no nome da comunidade:

- Angola Open Source Community,
- Angola OpenSource Community, e  
- Angola Open-Source Community

recomendo escolher uma dessas versões e ajustar todas as ocorrências, para fortalecer a marca.

**Comment by antonio-pedro99** [comment]
> No site, repositório e organização do GitHub vejo diferentes ocorrências desse termo no nome da comunidade:
> 
> * Angola Open Source Community,
> * Angola OpenSource Community, e
> * Angola Open-Source Community
> 
> recomendo escolher uma dessas versões e ajustar todas as ocorrências, para fortalecer a marca.

Olá  @dellagustin, muito obrigado pelos reparos.
A Terminologia que usamos é Angola Open-source Community.

**Comment by antonio-pedro99** [comment]
Conforme discutimos, mudamos todas ocorrências para Angola Open-source Community na PR #75.
@dellagustin obrigado pela sugestão :)
